content/md/retailer-onboarder/first-steps/terms-and-conditions.md

```diff
@@ -0,0 +1,35 @@
+---
+id: terms-and-conditions
+themes: 
+title: Terms and conditions
+popular: false
+---
+
+# Privacy Policy for the Use of Ethn.io in Akeneo SaaS Products
```

Comment:
```suggestion
# Privacy Policy for the Use of Ethn.io in Onboarder
```

Reply:
@PierrickMartos will it only apply for Onboarder?

Reply:
No but since we are into the Onboarder section I thought it was better